src/main/java/io/connect/scylladb/ComposeAddressTranslator.java

```diff
@@ -0,0 +1,70 @@
+package io.connect.scylladb;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.policies.AddressTranslator;
+import java.net.InetSocketAddress;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import org.json.*;
+
+/**
+ *
+ * @author dj@compose.com
```

Comment:
??

Reply:
that  shouldn't be there.

Reply:
Who's the author of this code? Do we have the right to publish it under the license of this project? Compose is an IBM company.

Reply:
We shouldn't be having any licence issue. Please refer to the below link.
https://github.com/compose-ex/java-composeaddresstranslator/blob/master/src/main/java/com/compose/scyllacompose/ComposeAddressTranslator.java

Reply:
That code you mentioned is under MIT license and is copyrighted by IBM:
```
// Copyright (c) 2016 Compose, an IBM company
//
// MIT License
```

We're publishing under Apache. I'm not an expert on licensing @penberg, @avikivity or @nyh could you help, please? Is it ok to copy code from MIT to Apache project and drop copyright notice?

Reply:
The MIT license is similar to the Apache license, with slightly less details on things you must do. It's fine to insert it into our software, even if it is Apache licensed or AGPL. But you definitely need to keep the copyright notice and license in the source code and can't just drop it.
If we were really serious, we should have also collected these copyright statements in a separate directory (e.g., scylla has a separate licenses/ directory) so we can also provide them with binary versions of Scylla, not just in the source code.